src/components/Containers/Messages/MessagesContent/ResultsSummary.js

```diff
@@ -1,24 +1,38 @@
-import React, { useContext } from 'react';
+import React, { useState, useContext } from 'react';
 import styled from 'styled-components';
 import { borderSeparator, standardPadding, colorPrimary } from '../../../../styles/styleVariables';
 
 // Context
 import { AccountContext } from '../MessagesMain';
 import formatNumber from '../../../../util/formatNumber';
 
+// Children
+import AccountExportModal from '../AccountExportModal';
+
 const ResultsSummary = () => {
   const { messages, messagesTotalCount } = useContext(AccountContext);
-
+  const [showExportModal, setShowExportModal] = useState(false);
   return (
-    <ResultsSummaryStyled>
-      <Summary>
-        <h3>
-          Displaying <span>{formatNumber(messages.length)}</span> of
-          {messagesTotalCount === 10000 ? ' more than ' : ' '}
-          {formatNumber(messagesTotalCount)} results
-        </h3>
-      </Summary>
-    </ResultsSummaryStyled>
+    <>
+      <ResultsSummaryStyled>
+        <Summary>
+          <h3>
+            Displaying <span>{formatNumber(messages.length)}</span> of
+            {messagesTotalCount === 10000 ? ' more than ' : ' '}
+            {formatNumber(messagesTotalCount)} results
+          </h3>
+        </Summary>
+        <Actions>
+          <p onClick={() => setShowExportModal(true)} data-cy="export-as-records-request-button">
```

Comment:
Is there a reason this isn't a button?

Reply:
Honestly, not a very good one-- it's just a pain in the ass to "de-style" the default button stylings for all the various user-agents. Definitely an accessibility no-no, the way I've got it here.